spec/rubocop/cop/style/inverse_methods_spec.rb

```diff
@@ -209,6 +195,32 @@ def test_method
     end
   end
 
+  it 'registers an offense for using `any?` method' do
+    expect_offense(<<~RUBY)
+      !nullable.any?(&:odd)
+      ^^^^^^^^^^^^^^^^^^^^^ Use `none?` instead of inverting `any?`.
+    RUBY
+  end
```

Comment:
`expect_correction` should be included for this test as well as the one on line 211.

Reply:
It was a redundant test, so it has been removed.